responsibleai/requirements.txt

```diff
@@ -1,4 +1,4 @@
-dice-ml>=0.10,<0.11
+dice-ml>=0.11,<0.12
```

Comment:
no need to fix here but the easiest way of saying that is
```
dice-ml~=0.11

Reply:
yeah, this can be addressed later.